java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -288,6 +289,43 @@ public void getdel(BaseClient client) {
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void getex(BaseClient client) {
+        String key1 = "{key}" + UUID.randomUUID();
+        String value1 = String.valueOf(UUID.randomUUID());
+        String key2 = "{key}" + UUID.randomUUID();
+
+        client.set(key1, value1).get();
+        String data = client.getex(key1).get();
+        assertEquals(data, value1);
```

Comment:
you should also check that client.ttl(key1) is >= 0.  (ie this ensures that it has an expiry).

Reply:
here you can add: 
```
assertEquals(-1, client.ttl(key1).get());
```